Z64Utils/F3DZEX/Render/Zelda/SkeletonRenderer.cs

```diff
@@ -100,20 +100,24 @@ public void SetAnim(AnimationHolder anim, Memory mem)
             SetFrame(mem, 0);
         }
 
-        public void SetSkeleton(SkeletonHolder skel, Memory mem)
+        public void SetSkeleton(SkeletonHolder skel, Memory mem, Func<int, SegmentedAddress, string> limbNamer = null)
         {
             _skel = skel;
 
-
-            byte[] limbsData = mem.ReadBytes(_skel.LimbsSeg, _skel.LimbCount * 4);
+            byte[] limbsData = mem.ReadBytes(_skel.LimbsSeg, _skel.LimbCount * 4);
 
             var limbs = new SkeletonLimbsHolder("limbs", limbsData);
 
             _limbs = new List<SkeletonLimbHolder>();
             for (int i = 0; i < limbs.LimbSegments.Length; i++)
             {
+                string limbName = null;
+                if (limbNamer != null)
+                    limbName = $"{i+1} " + limbNamer(i, limbs.LimbSegments[i]);
+                if (limbName == null)
+                    limbName = $"limb_{i+1}";
```

Comment:
That `if` statement is a bit confusing IMO.
```suggestion
                string limbName = limbNamer != null
                    ? $"{i+1}: {limbNamer(i, limbs.LimbSegments[i])}"
                    : $"limb_{i+1}";
```

Reply:
Oh thanks for noticing that, it's actually a mistake, I mean for limbNamer to be allowed to return null to have the name be a default

I will also try to find a better name for `limbNamer`